book/src/secrets-and-credentials.md

```diff
@@ -48,9 +48,20 @@ The value of both parameters may be a literal or a template containing the follo
 
     * `${pv.name}`
       * Automatically replaced with the name of the `PersistentVolume` object being provisioned at provision.
+    * `${pvc.namespace}`
+      * Automatically replaced with the namespace of the `PersistentVolumeClaim` object being provisione
+
+The value of `csi.storage.k8s.io/provisioner-secret-name` also supports the following template variables which are automatically replaced by the `external-provisioner` at provision time:
+
+  * `${pvc.name}`
+    * Automatically replaced with the name of the `PersistentVolumeClaim` object being provisioned.
 
 If specified, the CSI `external-provisioner` will attempt to fetch the secret before provisioning and deletion.
 
+If the entire namespace was deleted, including the secret needed for deletion, then no secret will be passed to the
+delete call. If this happens when the driver requires a secret for deletion, then the volume and PV may need to be
+manually cleaned up. 
+
 If no such secret exists in the Kubernetes API, or the provisioner is unable to fetch it, the provision or delete operation fails.
```

Comment:
This sentence also has to be updated :-)

Reply:
Updated this and moved the sentence about manually cleaning up to be after this.